tests/test_oic_provider_logout.py

```diff
@@ -779,9 +804,9 @@ def test_do_verified_logout_just_the_one(self):
         # I know the session backend DB is a DictSessionBackend so I can use that
         _sid = list(self.provider.sdb._db.storage.keys())[0]
 
-        self.provider.httpc = DummyHTTPClient(200)
-
+        # There is no back channel logout, hence no HTTP POST and no need for responses
```

Comment:
Then it might be a good idea to add responses anyway as they will make sure that no communication is done. This makes this behaviour fixed as the test would otherwise fail.

Reply:
OK